.github/workflows/psm-interop.yaml

```diff
@@ -0,0 +1,74 @@
+name: PSM Interop
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'v1.*'
+
+permissions:
+  contents: read
+
+jobs:
+  unittest:
+    # By default, only version is printed out in parens, e.g. "unittest (3.10)"
+    # This changes it to "unittest (python3.10)"
+    name: "unittest (python${{ matrix.python_version }})"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python_version: ["3.9", "3.10"]
```

Comment:
Not blocking, but since there are now multiple places where we're encoding the supported versions for the interop test framework, we might want to dedupe via (for example) templating.

Reply:
Agree, but at the moment the manual burden of updating these once in a while is just cheaper/faster than templating it. Long term, I'll hoping to switch to bazel.